domains/louis.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A portofolio",
+    "repo": "https://github.com/louloudev59/portofolio",
+    "owner": {
+        "username": "<louloudev59>",
+        "email": "louis@gmail.com",
+        "twitter": "<no-twitter>"
+    },
+    "record": {
+        "CNAME": "https://louloudev59.github.io/portofolio/"
```

Comment:
Invalid CNAME, must be a single valid hostname.

Reply:
Ok, I try to fix that soon, thanks !

Reply:
Good, I fix that